docs/user-guide/configuration/annotations.md

```diff
@@ -33,7 +33,7 @@ Some Annotations are used to modify the properties of the build environment. The
 | Annotation | Values | Description |
 |------------|--------|-------------|
 | beta.screwdriver.cd/executor | Ask your cluster admin | This will determine what compute system is used to run the build. For example, set the build to run in a VM, a kubernetes pod, a docker container, or a Jenkins agent. |
-| beta.screwdriver.cd/cpu | `LOW` / `HIGH` | When using a `k8s-vm` executor, this will allow the user to choose between 2 CPU resources (`LOW`) and 6 CPU resources (`HIGH`). Default is `LOW`. |
-| beta.screwdriver.cd/ram | `LOW` / `HIGH` | When using a `k8s-vm` executor, this will allow the user to choose between 2 GB RAM (`LOW`) and 12 GB RAM (`HIGH`). Default is `LOW`. |
+| beta.screwdriver.cd/cpu | `MICRO` / `LOW` / `HIGH` | When using a `k8s-vm` executor, this will allow the user to choose between 0.5 (`MICRO`), 2 (`LOW`) and 6 (`HIGH`) CPU resources. Default is `LOW`. |
```

Comment:
>  When using a `k8s-vm` executor,

We can also use these annotations with `k8s` executor so we should also add the description.

Reply:
I add `k8s` description. Thank you!

Reply:
I think we may not add lines, just add the sentence of `k8s` executor to existing lines. it looks duplicate.

Reply:
I understood. I fixed.